### PR TITLE
add: grid component

### PR DESCRIPTION
**Pull request by devsurajrai**
Added the grid component to the library. Kindly review.

grid

### CODE REVIEW

File: components/grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+.grid-two{
+    display:grid;
+    grid-template-columns: repeat(2,1fr);
+    width:80%;
+    height:80%;
+    background-color:var(--secondary);
+    place-items:center;
+
+}
+/* grid having three items  */
```
Comment (vaishnavipaliya11) [inline components/grid/grid.css:14]: Liked the way  you mention specific comment for understanding.

**Review by vaishnavipaliya11 — APPROVED** [review]
code looks clean and easy to understand. 
It's good how you reused your utility classes.
You can move ahead.

**Review by Dev-AkshatGupta — APPROVED** [review]
Nothing to suggest in the grid component nice that you have used a repeat in grid great work man

File: components/grid/grid.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+                <iframe class="iframe-codes"
+                    src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522grid-two%2520p-md%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522item%2522%253EItem%25201%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522item%2522%253EItem%25202%253C%252Fspan%253E%250A%253C%252Fdiv%253E"
+                    style="width: 471px; height: 260px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+            <h2>Grid-3</h2>
+            <p class="text-sm">
+                For having two items in a grid simply copy the code below and paste it in your project.
```
Comment (saifudinbadani) [inline components/grid/grid.html:51]: I think you meant three items here, otherwise  rest of it is okay.

**Review by saifudinbadani — APPROVED** [review]

File: components/grid/grid.html (hunk 2 above)
Comment (devsurajrai) [reply components/grid/grid.html:51]: will change that !

**Pull request merged**